funfact/loss.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import jax.numpy as np
```

Comment:
Now we can replace line with the backend import. 😎

Reply:
Done.